client/src/services/ApiClient.js

```diff
@@ -116,9 +116,19 @@ export async function getEmployeeByID(employeeID) {
 // Client
 // ------
 
-export async function findClients(searchString) {
+export async function findClients(searchString, hasPendingCancelation) {
   let path = "/api/clients";
-  if (searchString) path += `?searchString=${searchString}`;
+
+  if(searchString || hasPendingCancelation!==null){
+    path += "?";
+    if(searchString){
+      path += `searchString=${searchString}&`;
+    }
+    if(hasPendingCancelation!==null){
+      path += `hasPendingCancelation=${hasPendingCancelation}&`;
+    }
+    path = path.substring(0, path.length-1);//delete the last character, that is &
```

Comment:
I think it works with this management of "&" inside the query. Try to update from main and test it (I've merged the BE). Otherwise use the pattern used for findProducts API